root/etc/e-smith/templates/etc/httpd/conf.d/proxypass.conf/20Default_virtualhost

```diff
@@ -0,0 +1,63 @@
+{
+use esmith::ConfigDB;
+my $db = esmith::ConfigDB->open_ro('proxypass');
+return "" unless $db;
+my @proxypass = $db->get_all_by_prop('type' => 'VhostReverse');
+if (@proxypass) {
+    $OUT .= <<'EOF';
+#
+# Default virtualhost
```

Comment:
proxypass.conf has priority over virtualhosts.conf. As we saw for "TLS policy" page, we must avoid to swap VirtualHost definition order because the first one is considered the "default", and matches any vhost name not listed in ServerName directives.

We probably do not need to re-define a "default" VirtualHost here. This fragment could be removed.